docs/oscar_references.bib

```diff
@@ -171,6 +171,18 @@ @Article{BDLPSS13
   doi           = {10.1016/j.jsc.2012.07.002}
 }
 
+@Article{BE23,
+  author        = {Brandhorst, Simon and Elkies, Noam D.},
+  title         = {Equations for a {K}3 {L}ehmer map},
```

Comment:
```suggestion
  title         = {Equations for a K3 Lehmer map},
```
DocumenterCitations.jl sometimes shows these braces here. But since it never changes casing, they are safe to remove